demos/tests/run_tests.py

```diff
@@ -52,6 +55,17 @@ def parse_args():
         help='Model Optimizer entry point script')
     return parser.parse_args()
 
+def parce_result(demo_name, device, pipline, execution_time):
```

Comment:
pls, fix typo: pipline should be pipeline

Reply:
`parce` -> `parse`